Makefile

```diff
@@ -5,8 +5,7 @@ TAG_VERSION?=$(shell git tag --points-at | sort -Vr | head -n1)
 CGO_LDFLAGS="-L 'gen/third_party/rplidar_sdk-release-${VERSION}/sdk/output/${OS}/Release/'"
 GO_BUILD_LDFLAGS = -ldflags "-X 'main.Version=${TAG_VERSION}' -X 'main.GitRevision=${GIT_REVISION}'"
 
-default: install-swig swig
-.PHONY: default
```

Comment:
We don't need this since we don't have a file named `default` in this directory.

Reply:
Would this mean that I would have to add `.PHONY: ...` above each of these make targets? Should I follow this style guide? https://clarkgrubb.com/makefile-style-guide#phony-targets